staging/src/k8s.io/apiextensions-apiserver/test/integration/change_test.go

```diff
@@ -73,9 +73,12 @@ func TestChangeCRD(t *testing.T) {
 			default:
 			}
 
+			time.Sleep(10 * time.Millisecond)
+
 			noxuDefinitionToUpdate, err := apiExtensionsClient.ApiextensionsV1beta1().CustomResourceDefinitions().Get(noxuDefinition.Name, metav1.GetOptions{})
 			if err != nil {
-				t.Fatal(err)
+				t.Error(err)
```

Comment:
remind me why `t.Fatal` was bad here?

Reply:
"the goroutine calls T.Fatal, which must be called in the same goroutine as the test"

Reply:
It should be this: "the goroutine calls T.Fatal, which must be called in the same goroutine as the test"

Reply:
Double checked.  You can try this command: 
`hack/verify-staticcheck.sh vendor/k8s.io/apiextensions-apiserver/test/integration`